### ISSUE TITLE
Useless node name in CLI output

### ISSUE THREAD

**Issue by tgraf** [cilium/hubble#86]
The node name in the output is pretty useless when operating the CLI in a single node context and only steals space.

```
ks exec -ti hubble-gt7m2 -- hubble observe -t drop --follow
Feb  9 13:28:21.965 [ip-192-168-12-253.us-west-2.compute.internal]: default/pod-to-a-l3-denied-cnp-65fd99db76-2knrs:51104 -> default/echo-a-7f74df756d-s4b9p:80(http) Policy denied (L3) DROPPED (TCP Flags: SYN)
Feb  9 13:28:22.798 [ip-192-168-12-253.us-west-2.compute.internal]: fe80::3042:1dff:fe0c:17df -> ff02::2 Unsupported L3 protocol DROPPED (ICMPv6 RouterSolicitation)
Feb  9 13:28:23.743 [ip-192-168-12-253.us-west-2.compute.internal]: default/pod-to-a-l3-denied-cnp-65fd99db76-2knrs:51112 -> default/echo-a-7f74df756d-s4b9p:80(http) Policy denied (L3) DROPPED (TCP Flags: SYN)
Feb  9 13:28:23.982 [ip-192-168-12-253.us-west-2.compute.internal]: default/pod-to-a-l3-denied-cnp-65fd99db76-2knrs:51104 -> default/echo-a-7f74df756d-s4b9p:80(http) Policy denied (L3) DROPPED (TCP Flags: SYN)
Feb  9 13:28:24.749 [ip-192-168-12-253.us-west-2.compute.internal]: default/pod-to-a-l3-denied-cnp-65fd99db76-2knrs:51112 -> default/echo-a-7f74df756d-s4b9p:80(http) Policy denied (L3) DROPPED (TCP Flags: SYN)
Feb  9 13:28:26.765 [ip-192-168-12-253.us-west-2.compute.internal]: default/pod-to-a-l3-denied-cnp-65fd99db76-2knrs:51112 -> default/echo-a-7f74df756d-s4b9p:80(http) Policy denied (L3) DROPPED (TCP Flags: SYN)
Feb  9 13:28:30.222 [ip-192-168-12-253.us-west-2.compute.internal]: fe80::3042:1dff:fe0c:17df -> ff02::2 Unsupported L3 protocol DROPPED (ICMPv6 RouterSolicitation)
```

**Comment by genbit** [comment]
@tgraf even in multi-node context, I find this information could be hidden behind a flag --node that will enable showing up node name.

**Comment by glibsm** [comment]
Is this only the case for the compact output, or also all other types? json/dict?

**Comment by genbit** [comment]
i think it's only the case for default hubble output (is it compact). JSON could have it. Not sure what is `dict`

**Comment by mdnix** [comment]
@tgraf Is this still relevant? I'd be happy to help

**Comment by Rolinh** [comment]
@glibsm This is the case for the compact and JSON outputs but I think we want to keep it in JSON output. Dict and tab output omit the node's name.

@mdnix Yes, still relevant. As @genbit suggested, it'd be nice to have a flag to display the node's name though.

**Comment by Rolinh** [comment]
Implemented in #358, closing.
